assets/styles/style.css

```diff
@@ -0,0 +1,149 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css");
+@import url("https://enigma-ni.netlify.app/src/css/enigma.css");
+
+@font-face {
+  font-family: "SAMAN";
+  src: url("/assets/fonts/SAMAN.eot");
+  src: url("/assets/fonts/SAMAN.eot") format("embedded-opentype"),
+    url("/assets/fonts/SAMAN.woff2") format("woff2"),
+    url("/assets/fonts/SAMAN.woff") format("woff"),
+    url("/assets/fonts/SAMAN.ttf") format("truetype"),
+    url("/assets/fonts/SAMAN.svg#SAMAN") format("svg");
+}
+
+header {
+  border-block-end: 1px solid var(--border-color);
+}
+
+.nav-brand {
+  font-family: "SAMAN";
+  font-size: 2rem;
+}
+
+footer {
+  border-block-start: 1px solid var(--border-color);
+}
+
+.sidebar {
+  inline-size: 15rem;
+}
+
+@media screen and (min-width: 40rem) {
+  .sidebar-visible-tablet-onwards {
+    display: block;
+  }
+}
+
+.home-grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(25rem, 1fr));
+  justify-items: center;
+  gap: 1.5rem;
+  margin-block-start: 1rem;
+}
+
+.home-grid-container .card-img {
+  max-inline-size: 20rem;
+  max-block-size: 25rem;
+}
+
+#auth-modal .modal-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: var(--spacing-16);
+  text-align: center;
+}
+
+#auth-modal .check-container {
+  color: var(--color-green-600);
```

Comment:
using color names as variables names is not good approach try to use descriptive name for that so that it will be easy to update in future with different color as per requirement.

Reply:
ok i will update this later